### ISSUE TITLE
AdGuard Extension ID for GPO

### ISSUE THREAD

**Issue by RBrenckle** [AdguardTeam/AdguardBrowserExtension#3073]
### Please answer the following questions for yourself before submitting a question.

- [X] I am running the latest version
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### Ask a question

We are looking to implement AdGuard in our environment and would like to do so via a GPO using the extension ID.  I have searched the knowledge base and reviewed the documentation for Adguard online but am not finding the ID.  Where do I find the extension ID so it can be added to the user community via a GPO?

Thank you.

**Comment by windwak3r** [comment]
@RBrenckle, hello! You can take the ID from Chrome Web Store.

This is how it looks like for the MV3 extension:
<details>
  <summary>Screenshot 1</summary>

  ![Screenshot 1](https://github.com/user-attachments/assets/8385328e-e985-4281-9fcb-2263916dcf5a)

</details>

And this is how it looks like with Update URL:
`bgnkhhnnamicmpeenaelnjfhikgbkllg;https://clients2.google.com/service/update2/crx`

I used it to install the extension via ExtensionInstallForcelist.

**Comment by RBrenckle** [comment]
Hello, 

Thank you for the information and extension ID.  I added this to our AD GPO and it did install successfully on the desktop however, upon launching Google Chrome the install completed and a new tab was open.  We would like to prevent this from happening for our user community.  How do we do that?

We use AD GPO to install our other extensions and need to do the same with the AdGuard extension.  I tried both "normal-installed" and "force-installed" modes and received the same result both times: the install was completed upon launching Google.

 "bgnkhhnnamicmpeenaelnjfhikgbkllg": { "installation_mode": "force_installed", "update_url": "https://clients2.google.com/service/update2/crx" },  

Thank you for the additional direction.
